ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/facade/DefaultAuditor.java

```diff
@@ -79,6 +79,9 @@ private boolean shouldAudit(AuditEntry logEntry)
         {
             return !filter.isWhitelisted(logEntry);
         }
+        catch(Exception ex) {
+        	return false;
```

Comment:
I think this should be the other way around - return true in this case.
If we cannot find any whitelist-info, the we assume that "whitelisting" is false -> meaning that shouldAudit() should return true.